backend/app.py

```diff
@@ -0,0 +1,31 @@
+from contextlib import asynccontextmanager
+
+from fastapi import FastAPI
+from models import User
+from database import SessionDep, create_db_and_tables
+
+
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    create_db_and_tables()
+    yield
+
+
+app = FastAPI(lifespan=lifespan)
+
+
+@app.post("/admin")
```

Comment:
This is fine for now, in a next step you could create a router for `users` and just add the routes to that router :)

Reply:
Do you mean user router is "/user" ?

Reply:
/admin route has been changed to /user